pkg/clients/client.go

```diff
@@ -88,8 +89,15 @@ func New(backendName, backendType string, backends map[string]map[string]config.
 
 		return redhatrover.NewClient(backend.Connection,
 			appConfig.HttpClient.ConnectionPoolConfig, appConfig.HttpClient.HystrixResiliencyConfig)
+	case "atlan":
+		appConfig, err := config.GetConfig()
+		if err != nil {
+			return nil, err
+		}
```

Comment:
wondering if we can extract this out from here and call this at the start of the function?